nncf/quantization/algorithms/bias_correction/algorithm.py

```diff
@@ -264,7 +242,13 @@ def traverse_to_input_layers(node, output):
         for stat_node in stats_nodes:
             input_nodes.update(nncf_graph.traverse_graph(stat_node, traverse_to_input_layers, traverse_forward=False))
 
-        return [input_node.node_name for input_node in input_nodes], [stat_node.node_name for stat_node in stats_nodes]
+        # TODO(andrey-churkin): Do we really need to store it as dict?
```

Comment:
Why can't we resolve this TODO in this PR?

Reply:
I removed this todo.